pkgs/desktops/lumina/lumina/default.nix

```diff
@@ -33,19 +33,19 @@ mkDerivation rec {
   ];
 
   buildInputs = [
-    xorg.libxcb
-    xorg.libXcursor
-    xorg.libXdamage
-    xorg.xcbutilwm
-    xorg.xcbutilimage
+    desktop-file-utils
```

Comment:
This one should likely go to `nativeBuildInputs`. OR do you need one of 

- desktop-file-edit
- desktop-file-install
- desktop-file-validate
- update-desktop-database

at runtime?

Reply:
@jtojnar In fact `desktop-file-utils` is really not needed. Removed it.